account/update_account_with_account_key_public/boilerplate.js

```diff
@@ -107,4 +107,5 @@ async function run() {
     const vtReceipt = await caver.rpc.klay.sendRawTransaction(vt)
     console.log(`Receipt of value transfer transaction after account update => `)
     console.log(vtReceipt)
+    console.log('You must replace SENDER_PRIVATE_KEY in `caver-js-exapmles/.env` with newly-updated private key to use the updated account continuously.')
```

Comment:
```suggestion
    console.log(`Since the AccountKey of your Klaytn account has been successfully changed, you need to replace \`SENDER_PRIVATE_KEY="${senderPrivateKey}"\` to \`SENDER_PRIVATE_KEY="${newKey}"\` in \`caver-js-exapmles/.env\` to use your updated account.`)
```